RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -685,9 +712,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      reco::MuonTime muonTime;
      reco::MuonTimeExtra dtTime;
      reco::MuonTimeExtra cscTime;
+     reco::MuonTime rpcTime;
      reco::MuonTimeExtra combinedTime;
 
-     theTimingFiller_->fillTiming(muon, dtTime, cscTime, combinedTime, iEvent, iSetup);
+     theTimingFiller_->fillTiming(muon, dtTime, cscTime, rpcTime, combinedTime, iEvent, iSetup);
```

Comment:
Yes, I saw it in the baseline as well.  I am confused, is there something wrong here?